docs/docusaurus/docs/core/introduction/try_gx_end_to_end.py

```diff
@@ -0,0 +1,190 @@
+"""
+This example script allows the user to try out GX by validating Expectations
+ against sample data.
+
+The snippet tags are used to insert the corresponding code into the
+ Great Expectations documentation.  They can be disregarded by anyone
+ reviewing this script.
+"""
+
+# <snippet name="docs/docusaurus/docs/core/introduction/try_gx_end_to_end.py full end2end script">
+
+# Import required modules from GX library.
+# <snippet name="docs/docusaurus/docs/core/introduction/try_gx_end_to_end.py import gx library">
+import great_expectations as gx
+import great_expectations.expectations as gxe
```

Comment:
To minimize extra imports, elsewhere in the docs we've stopped using `import great_expectations.expectations as gxe`.  Instead, when Expectations are created we're using `gx.expectations.ExpectWhatever()` instead of `gxe.ExpectWhatever()`.

Reply:
I'll update to just import `great_expectations` and use the `gx` alias. Thanks for the heads up!